src/vault.py

```diff
@@ -353,7 +355,7 @@ def cleanup(args):
     # Cleanup decrypted files
     yaml_file = args.yaml_file
     try:
-        os.remove(f"{yaml_file}.dec")
+        os.remove(f"{yaml_file}.dec")  # TODO per env? or blanket clean yaml_file.*.dec
```

Comment:
hey what are your thoughts on the `clean` operation?  

I would probably lean towards keeping it the same as encode and decode... so add a flag, and then delete the specific file based on the input _but_ I think an argument can be made for simply cleaning all decoded instances of a given file.

Thoughts?

Reply:
I think keeping the logic the same as the rest of the commands would be a good idea. I'm not sure how often people use the clean command, it's just there for feature parity with another utility.